manual/manual_circuit.rst

```diff
@@ -482,28 +507,35 @@ Another notable example that is common to many algorithms and high-level circuit
     circ.add_pauliexpbox(PauliExpBox([Pauli.X, Pauli.Y, Pauli.Y, Pauli.Y], 0.2), [0, 1, 2, 3])
     circ.add_pauliexpbox(PauliExpBox([Pauli.Y, Pauli.X, Pauli.Y, Pauli.Y], -0.2), [0, 1, 2, 3])
 
-Statevectors and Unitaries
---------------------------
+In addition to the box types mentioned above ``pytket`` also supports a :py:class:`ToffoliBox` structure. :py:class:`ToffoliBox` can be used to prepare an arbitrary permutation of the computational basis states using. Within the box this permutation is performed using the classical reversible gateset :math:`\{X,CnX\}`.
+
+In order to construct a :py:class:`ToffoliBox` the user must supply the desired permutation as a dictionary specifying the action of the box on different input states. If no key:value pair is specified for a given basis state then the mapping is just the identity map and the state is unchanged by the :py:class:`ToffoliBox`. 
```

Comment:
Rewrite to be something more like this (my formatting is probably off in github!):
different input basis states, where a key:value pair given as `List[Bool]` implies that the basis state key should be mapped to the basis state value. The given dictionary should provide permutations that correspond to complete cycles of basis states, i.e. if a basis state is present as a key then it must also be present as a value in the dictionary. At construction the permutations are translated to cycles and an error is thrown if the provided permutation is invalid.

Reply:
Thanks, I've rewrote this a bit - let me know if further changes are needed.